hubspot/tasks.py

```diff
@@ -44,3 +50,37 @@ def sync_line_item_with_hubspot(line_id):
     body = [make_line_item_sync_message(line_id)]
     response = send_hubspot_request("LINE_ITEM", HUBSPOT_SYNC_URL, "PUT", body=body)
     response.raise_for_status()
+
+
+@app.task
+def check_hubspot_api_errors():
+    """Check for and log any errors that occurred since the last time this was run"""
+    offset = 0
+    last_check, _ = HubspotErrorCheck.objects.get_or_create(
+        defaults={"checked_on": now_in_utc()}
+    )
+    last_timestamp = int(last_check.checked_on.timestamp() * 1000)
```

Comment:
Can you create a utility method for this? `convert_to_hubspot_timestamp` or something like that?